pandas/core/internals/blocks.py

```diff
@@ -1163,7 +1160,6 @@ def where(self, other, cond, errors="raise") -> list[Block]:
         assert cond.ndim == self.ndim
         assert not isinstance(other, (ABCIndex, ABCSeries, ABCDataFrame))
 
-        assert errors in ["raise", "ignore"]
```

Comment:
we are no longer raising on invalid `errors` value, but this was already not a correct user facing ValueError. so maybe just removing is fine and allowing any  value/type for errors is ok?

Reply:
you mean remove this line but not do the rest of this PR?

Reply:
I mean that if we wanted to retain the validation it would now need to be higher up the stack but also probably ok that the errors parameter is no longer validated?